app/carbonzipper/metrics.go

```diff
@@ -189,6 +190,17 @@ func NewPrometheusMetrics(config cfg.Zipper) *PrometheusMetrics {
 					config.Monitoring.FindDurationLin.BucketsNum),
 			},
 		),
+		FindDurationByBackend: prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name: "find_duration_seconds_by_backend",
```

Comment:
I think unit part (`_seconds`) usually go last, this would also make metric name closer to struct field name. Though not sure what's the pattern in carbonzipper, if any.

Reply:
ok seems like surrounding code puts it also in the middle..